### PR TITLE
Add stub for pathlib2.Path.__div__

### PR DESCRIPTION
**Pull request by li-dan**

### CODE REVIEW

File: stdlib/3/pathlib.pyi [hunk 1]
```diff
@@ -31,6 +31,8 @@ class PurePath(_PurePathBase):
     def __gt__(self, other: PurePath) -> bool: ...
     def __ge__(self, other: PurePath) -> bool: ...
     def __truediv__(self: _P, key: Union[str, PurePath]) -> _P: ...
+    if sys.version_info[0] == 2:
```
Comment (srittau) [inline stdlib/3/pathlib.pyi]: Minor nit: In typeshed we use the form `sys.version_info < (3,)` (or similar) consistently. I would like to keep it this way.

File: third_party/2/pathlib2.pyi [hunk 2]
```diff
@@ -31,6 +31,8 @@ class PurePath(_PurePathBase):
     def __gt__(self, other: PurePath) -> bool: ...
     def __ge__(self, other: PurePath) -> bool: ...
     def __truediv__(self: _P, key: Union[str, PurePath]) -> _P: ...
+    if sys.version_info[0] == 2:
```
Comment (srittau) [inline third_party/2/pathlib2.pyi]: Ditto.

**Comment by li-dan** [comment]
Fixed.

File: stdlib/3/pathlib.pyi [hunk 3]
```diff
@@ -31,7 +31,7 @@ class PurePath(_PurePathBase):
     def __gt__(self, other: PurePath) -> bool: ...
     def __ge__(self, other: PurePath) -> bool: ...
     def __truediv__(self: _P, key: Union[str, PurePath]) -> _P: ...
-    if sys.version_info[0] == 2:
+    if sys.version_info[0] < (3,):
```
Comment (srittau) [inline stdlib/3/pathlib.pyi]: Nearly, the `[0]` must be removed.

**Comment by li-dan** [comment]
Ah, I should've run the tests.

**Pull request merged**